test/helper/config.lua

```diff
@@ -6,7 +6,10 @@ require('json').cfg { encode_use_tostring = true }
 local config = {}
 
 config.root = fio.dirname(fio.abspath(package.search('init')))
+package.setsearchroot(config.root)
```

Comment:
Users of tarantool community 1.10 can't run this tests

Reply:
Oh, didn't think about it. Thank you.